src/compile/legend.ts

```diff
@@ -29,25 +30,18 @@ export function parseLegendComponent(model: UnitModel): Dict<VgLegend> {
 }
 
 function getLegendDefWithScale(model: UnitModel, channel: Channel): VgLegend {
+  // use a special scale so we can overrride the mark props and labels
```

Comment:
"For binned field with continuous scale, use a ... "

(The current comment misleads that we always use...)